components/statistics/count.lua

```diff
@@ -230,6 +230,11 @@ function Count._baseConditions(args, isTournament)
 		startDateKey = 'startdate'
 		endDateKey = 'enddate'
 		sortDateKey = 'sortdate'
+
+		if not Logic.readBool(args.includeAllStatus) then
+			conditions:add{ConditionNode(ColumnName('status'), Comparator.neq, 'cancelled')}
+			conditions:add{ConditionNode(ColumnName('status'), Comparator.neq, 'postponed')}
```

Comment:
```suggestion
			conditions:add{ConditionNode(ColumnName('status'), Comparator.eq, '')}
```

Reply:
Like this @iMarbot

Reply:
https://liquipedia.net/commons/index.php?title=Module:StatisticsPortal/dev&action=edit#mw-ce-l663

I've done something similar so it should work, I think

Reply:
will cause issues on some wikis
some set `finished` as status for finished events (for being able to better check what is ongoing/recent)

Reply:
I will leave as is then